locale/mr.js

```diff
@@ -1,6 +1,7 @@
 //! moment.js locale configuration
```

Comment:
Shouldn't this file be removed from the Pull Request?
https://github.com/moment/moment/blob/develop/CONTRIBUTING.md#read-before-submitting-pull-requests
Point 3 says, do NOT submit locale/*.js